samples/addons/prometheus_recording_istio.yaml

```diff
@@ -299,7 +299,6 @@ data:
         source_labels:
         - __meta_kubernetes_pod_node_name
         target_label: node
-      - action: replace
       metric_relabel_configs:
```

Comment:
@delavet This is to replace app:kmeh to app:ztunnel? But i donot see we report metric with label `app:kmesh`

Reply:
https://github.com/kmesh-net/kmesh/blob/ab83d8ccffaca860a5a657d7873afddd5327a122/samples/addons/prometheus_recording_istio.yaml#L245

this job scrape metrics for pods with `prometheus.io/scrape` annotations (kmesh pods have this), and it has a `label_map` config to convert pod labels to metrics labels.

https://github.com/kmesh-net/kmesh/blob/ab83d8ccffaca860a5a657d7873afddd5327a122/samples/addons/prometheus_recording_istio.yaml#L50

this job scrape metrics for pods with `app: kmesh` labels (kmesh pods also have this), this does not have `label_map` config.

The result is this prometheus configuration result in scraping kmesh metrics twice, one with `app:kmesh` metrics label (and be relabeld to `app:ztunnel`), one does not.